### PR TITLE
Create default_template.md

### PR DESCRIPTION
**Pull request by Chris-Schnaufer**
First cut of a default issue creation template

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/default_template.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+*Fill out this issue by completing the items below. Not every item may be relevent to you, so feel free to make changes*
```
Comment (dlebauer) [inline .github/ISSUE_TEMPLATE/default_template.md]: Can comment this out w/ html tags:

```suggestion
<!-- Fill out this issue by completing the items below. Not every item may be relevent to you, so feel free to make changes-->
```

File: .github/ISSUE_TEMPLATE/default_template.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+#### Add other supporting information that may be useful
+
+Versions, OS information, other:
+
+#### Completion criteria
+
+This is done when:
+- [ ] (replace this text)
+- [ ] documentation updated
```
Comment (dlebauer) [inline .github/ISSUE_TEMPLATE/default_template.md:23]: Added tests + docs to completion criteria.

**Review by dlebauer — APPROVED** [review]
Looks good.

I just added some space b/w section header and text

File: .github/ISSUE_TEMPLATE/default_template.md (hunk 2 above)
Comment (Chris-Schnaufer) [reply .github/ISSUE_TEMPLATE/default_template.md:23]: Thanks for the change

**Pull request merged**